client/packages/common/src/ui/components/domain/Dashboard/Widget.tsx

```diff
@@ -17,7 +17,7 @@ export const Widget: React.FC<PropsWithChildren<WidgetProps>> = ({
   height = '100%',
   title,
 }) => (
-  <Paper
+  <Card
```

Comment:
I've changed to a card here to avoid the Paper default 100% height property. That felt more clean than overriding Paper defaults, unless anyone is particularly attached to Paper?